go/def.bzl

```diff
@@ -625,8 +630,33 @@ go_env_attrs = {
         executable = True,
         single_file = True,
     ),
+    "_test_generator": attr.label(
+        executable = True,
+        default = Label("//go/tools:generate_test_main"),
+        cfg = "host",
+    ),
+    "_extract_package": attr.label(
+        default = Label("//go/tools/extract_package"),
+        executable = True,
+        cfg = "host",
+    ),
 }
 
+
+def get_go_toolchain(ctx):
```

Comment:
Can this be private? Looks like it isn't used outside of def.bzl right now, but maybe later?